orchestrator/db/models.py

```diff
@@ -478,21 +476,6 @@ class SubscriptionTable(BaseModel):
         deferred=True,
     )
 
-    port_mode = column_property(
```

Comment:
The two above should also be moved. They are a backwards compat thing and should not be used